include/backend/cuda_backend.h

```diff
@@ -1,20 +1,26 @@
 #pragma once
 
-#include <sycl/sycl.hpp>
-
 #include "backend/operations.h"
 #include "backend/type.h"
+#include "ranges.h"
 
 namespace celerity::detail::backend_detail {
```

Comment:
General readability question: do we actually need to qualify `celerity::range` and `celerity::id` in these files / inside the celerity namespace?

Reply:
I removed all unnecessary (and semantically not interesting) namespace qualifications on these types.